### PR TITLE
feat: update email templates

### PR DESCRIPTION
**Pull request by braianj**

### CODE REVIEW

File: inbox/src/adapters/email-renderer.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export async function createEmailRenderer(): Promise<IEmailRenderer> {
   async function renderEmail(template: InboxTemplates, to: string, context: any): Promise<Email> {
     return {
       to,
-      subject: templates[template][TemplatePart.SUBJECT](context),
+      ...JSON.parse(templates[template][TemplatePart.SUBJECT](context)),
```
[the remark this answers is not in the document]
Comment (braianj) [reply inbox/src/adapters/email-renderer.ts:48]: We are getting more than the subject

File: inbox/src/adapters/page-templates/unsubscription-all.handlebars [hunk 2]
```diff
@@ -1,32 +1,137 @@
...
+      <td>
+      <Table>
+          
+          {{!-- <tr>
+            <td>
+              <img
+                src="http://cdn.mcauto-images-production.sendgrid.net/2322095804444b83/b3f64b49-0c9e-4c53-bdab-60c3fa3c862b/600x240.png"
+                alt="Unsubscribe Banner" style="width: 100%; max-width: 600px; border-radius: 8px;" />
+            </td>
+          </tr> --}}
```
[the remark this answers is not in the document]
Comment (braianj) [reply inbox/src/adapters/page-templates/unsubscription-all.handlebars:81]: We need to re add that banner, I was waiting the new banner. I'll update it

File: inbox/src/controllers/handlers/unconfirmed-email-handlers.ts [hunk 3]
```diff
@@ -37,10 +40,25 @@ export async function storeUnconfirmedEmailHandler(
     const accountBaseUrl = await config.requireString('ACCOUNT_BASE_URL')
     const code = makeId(CODE_LENGTH)
     await db.saveUnconfirmedEmail(address, body.email, code)
+
+    let userName
+
+    try {
```
Comment (aleortega) [inline inbox/src/controllers/handlers/unconfirmed-email-handlers.ts:46]: No longer needed, right?

File: inbox/src/controllers/handlers/unconfirmed-email-handlers.ts [hunk 4]
```diff
@@ -37,10 +40,25 @@ export async function storeUnconfirmedEmailHandler(
...
+    } catch (error) {
+      logger.warn(`Error getting profile ${error}`)
+    }
+
     const email: Sendable = {
       ...(await emailRenderer.renderEmail(InboxTemplates.VALIDATE_EMAIL, body.email, {
         validateButtonLink: `${accountBaseUrl}/confirm-email/${code}`,
-        validateButtonText: 'Click Here to Confirm Your Email'
+        validateButtonText: 'Click Here to Confirm Your Email',
+        userName,
```
Comment (aleortega) [inline inbox/src/controllers/handlers/unconfirmed-email-handlers.ts:60]: What would happen if userName is null?

File: inbox/src/controllers/handlers/unsubscription-handlers.ts [hunk 5]
```diff
@@ -22,6 +25,18 @@ export async function unsubscribeAllHandler(
   subscription.details.ignore_all_email = true
   await db.saveSubscriptionDetails(address, subscription.details)
 
+  let userName
+
+  try {
```
Comment (aleortega) [inline inbox/src/controllers/handlers/unsubscription-handlers.ts:30]: Also not needed any more, right?

**Review by aleortega — APPROVED** [review]

File: inbox/src/controllers/handlers/unconfirmed-email-handlers.ts (hunk 4 above)
Comment (braianj) [reply inbox/src/controllers/handlers/unconfirmed-email-handlers.ts:60]: it wont show it, but that's ok

**Pull request merged**